README.md

```diff
@@ -70,7 +72,45 @@ public final class MyClassWithTracing {
 
 ### Hello "OpenCensus" stats events
 
-TODO
+Here's an example on defining TagKey, Measure and View, registering view, recording stats and get
+ViewData. For the complete example, see
```

Comment:
Suggestion: It might be clearer to put the steps in a bulleted list and avoid two levels of commas.

Reply:
Done.